src/ts/component/block/featured.tsx

```diff
@@ -5,6 +5,7 @@ import { ObjectType, Cell } from 'Component';
 import { I, C, UtilData, UtilCommon, UtilObject, UtilDate, Preview, focus, analytics, Relation, Onboarding, history as historyPopup, keyboard, translate } from 'Lib';
 import { blockStore, detailStore, dbStore, menuStore, commonStore } from 'Store';
 import Constant from 'json/constant.json';
+import { observable } from 'mobx';
```

Comment:
Move import from mobx to library imports, they usually are at the start of the document